operator/cmd/mesh/install.go

```diff
@@ -337,5 +339,5 @@ func getProfileAndEnabledComponents(setOverlay []string, inFilenames []string, f
 			}
 		}
 	}
-	return profile, enabledComponents, nil
+	return profile, v1alpha12.Namespace(iop.Spec), enabledComponents, nil
```

Comment:
Do we need to change to `controller.IstioNamespace` if v1alpha12.Namespace(iop.Spec) is ""?

Reply:
+1 to add guard for the empty namespace returned by `v1alpha12.Namespace(iop.Spec)`

Reply:
+1 great comment!!  Fixed